modules/sdk-coin-avaxp/src/lib/utils.ts

```diff
@@ -274,18 +269,29 @@ export class Utils implements BaseUtils {
    * @param network required to stringify addresses
    * @return mapper function
    */
-  mapOutputToEntry(network: AvalancheNetwork): (TransferableOutput) => Entry {
-    return (output: TransferableOutput) => {
-      const amountOutput = output.getOutput() as any as AmountOutput;
-      const address = amountOutput
-        .getAddresses()
-        .map((a) => this.addressToString(network.hrp, network.alias, a))
-        .sort()
-        .join('~');
-      return {
-        value: amountOutput.getAmount().toString(),
-        address,
-      };
+  mapOutputToEntry(network: AvalancheNetwork): (Output) => Entry {
+    return (output: Output) => {
+      if ('getOutput' in output) {
```

Comment:
this is hacky. can we have a better typing for this? 

maybe something like:
```
// https://www.typescriptlang.org/docs/handbook/advanced-types.html
private static isTransferableOutput(output: Output | TransferableOutput): output is TransferableOutput {
    return (output as TransferableOutput).getOutput !== undefined;

mapOutputToEntry(network: AvalancheNetwork): (Output | TransferableOutput) => Entry {
    return (output: Output | TransferableOutput) => {
        if (isTransferableOutput) {
            ...
        } else {
            ...
```

Reply:
done